src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,44 @@
 import React from 'react';
+import './PostInfo.scss';
+import { FullPost } from '../../types/Post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: FullPost,
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
+          { user && <UserInfo user={user} />}
+        </p>
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      {(comments.length > 0)
+        ? (<CommentList comments={comments} />)
```

Comment:
```suggestion
      {comments.length
        ? (<CommentList comments={comments} />)
```

Reply:
in my opinion, it's better to check with > 0 for length

Reply:
sometimes you could see 0 on page when you check for `comments.length` for example 
so in my opinion you could do variables const `hasComment = comments.length > 0 ` and use it
or use Boolean instead 
<img width="817" alt="Screenshot 2023-01-30 at 11 18 59" src="https://user-images.githubusercontent.com/94292654/215436753-65c74137-64ba-48d9-9e43-7e8889fbcf7b.png">

Reply:
@PolinaSukhorukova @NVyrsta